src/test/scala/chiselTests/StrongEnum.scala

```diff
@@ -575,11 +575,10 @@ class StrongEnumAnnotator extends Module {
   val indexed2 = vec_of_bundles(cycle)
 }
 
-@chiselName
 class StrongEnumAnnotatorWithChiselName extends Module {
   import EnumExample._
 
-  object LocalEnum extends ChiselEnum {
+  object LocalEnum extends ChiselEnum with AffectsChiselPrefix {
```

Comment:
should `ChiselEnum` `AffectsChiselPrefix`?

Reply:
ah i see what you mean, agree that maybe making ChiselEnums always be named is OK to do later